hand-written.md

```diff
@@ -41,38 +34,6 @@ Scala 2.12 enables the "GenBCode" backend by default.
 
 The new backend is more efficient than the default backend of Scala 2.11, because it directly generates ASM bytecode from Scala compiler trees, while the previous backend used an intermediate representation called "ICode".
```

Comment:
> The new backend is more efficient than the default backend

it isn't clear here whether the claim is that the new back end runs faster, or whether it generates faster code. I assume the former, so I'd suggest: "The new backend is faster than the old one." 

(note also it's inappropriate now to refer to the "default backend" since the new one is now the default.)